content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -10,7 +10,7 @@ weight: 80
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.21" state="stable" >}}
+{{< feature-state for_k8s_version="v1.22" state="stable" >}}
```

Comment:
For this line, the original document is correct: the CronJob API was stabilized in Kubernetes v1.21

Between Kubernetes v1.21 and v1.22, the semantics of the `.spec.schedule` field changed.
(I'm not sure how that aligns with Kubernetes API stability guarantees, and will raise an issue about this).

Perhaps we can add an extra note about the format of `.spec.schedule` before Kubernetes v1.22?

Reply:
Actually, the semantics only changed if you set the feature flag `CronJobControllerV2` to false whilst running v1.21.

Reply:
Thanks for your doubts!